.github/workflows/release.yml

```diff
@@ -299,7 +299,7 @@ jobs:
       build-linux-amd64-artifacts,
       build-linux-arm64-artifacts,
     ]
-    runs-on: ubuntu-2004-16-cores
+    runs-on: ubuntu-latest
```

Comment:
We need `ubuntu-22.04` to build the image? Fix the following start error:
```
greptime: /lib/aarch64-linux-gnu/libm.so.6: version `GLIBC_2.38' not found (required by greptime)
greptime: /lib/aarch64-linux-gnu/libc.so.6: version `GLIBC_2.39' not found (required by greptime)
```

Reply:
Same as: `dev-build.yml`, `nightly-build.yml`, `release-dev-builder-images.yaml`

Reply:
I see. Let me fix this.

Reply:
I saw this task is called `release-images-to-dockerhub`, does it actually build anything in this task?

Reply:
it's OK